pyroomacoustics/libroom_src/libroom.cpp

```diff
@@ -114,5 +131,37 @@ PYBIND11_MODULE(libroom, m) {
 
     m.def("area_2d_polygon", &area_2d_polygon,
         "Compute the signed area of a planar polygon");
+		
+	m.def("angle_between", &angle_between,
+		"Computes the angle between two 2D or 3D vectors");
+	
+	m.def("dist_line_point", &dist_line_point,
+		"Computes the distance between a point and an infinite line");
+	
+	
+	// Routines for the utility packages
+	m.def("equation", &equation,
+		"Computes the a and b coefficients in the expression y=ax+b given two points lying on that line.");
+		
+	m.def("compute_segment_end", &compute_segment_end,
+		"Computes the end point of a segment given the start point, the length, and the orientation");
+		
+	m.def("compute_reflected_end", &compute_reflected_end,
+		"This function operates when we know the vector [start, hit_point]. This function computes the end point E so that [hit_point, E] is the reflected vector of [start, hit_point] with the correct magnitude");
+	
+	m.def("intersects_mic", &intersects_mic,
+		"Determines if a segment intersects the microphone of specified center and radius");
+		
+	m.def("solve_quad", &solve_quad,
+		"Solves the quadratic system and outputs real roots");
+		
+	m.def("mic_intersection", &mic_intersection,
+		"Computes the intersection point between the ray and the microphone");
+		
+		
+	// Routines for the ray_tracing package
+	m.def("bilbo", &bilbo,"Computes the bilbo");
```

Comment:
We can remove this if it isn't needed anymore.

Reply:
Removed